src/Compilers/Core/Portable/CodeAnalysisResources.resx

```diff
@@ -571,7 +571,7 @@
     <value>If tuple element locations are specified, the number of locations must match the cardinality of the tuple.</value>
   </data>
   <data name="TuplesNeedAtLeastOneElement" xml:space="preserve">
-    <value>Tuples must have at least one element.</value>
+    <value>TupleTypeSymbol elementTypes must not be empty.</value>
```

Comment:
Does ```TupleTypeSymbol``` refer to a name of any public type? Otherwise, consider changing to ```Tuple type symbol```, or simply removing the word from the message - "The elementTypes must not be empty."